src/index.js

```diff
@@ -0,0 +1,147 @@
+    weatherGardenDisplay();  
+};
+
+// Function to decrease the temperature
+const temperatureDecrease = () => {
+    state.temperature -= 1; 
+    tempValue.textContent = `${state.temperature}°F`;  
+    temperatureColor();  
+    weatherGardenDisplay();  
+};
+
+const temperatureColor = () => {
+    tempValue.classList.remove("red", "orange", "yellow", "green", "teal");
+
+    if (state.temperature >= 80) {
+        tempValue.classList.add("red");
+    } else if (state.temperature >= 70) {
+        tempValue.classList.add("orange");
+    } else if (state.temperature >= 60) {
+        tempValue.classList.add("yellow");
+    } else if (state.temperature >= 50) {
+        tempValue.classList.add("green");
+    } else {
+        tempValue.classList.add("teal");
+    }
+}
+
+const weatherGardenDisplay = () => {
+    gardenSection.classList.remove("sunny", "cloudy", "rainy", "snowy");
+    if (state.temperature >= 80) {
+        gardenSection.classList.add("sunny")
+        landscape.innerHTML = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂"
+    } else if (state.temperature >= 70) {
+        gardenSection.classList.add("cloudy")
+        landscape.innerHTML = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷"
+    } else if (state.temperature >= 60) {
+        gardenSection.classList.add("rainy")
+        landscape.innerHTML = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃"
+    } else {
+        gardenSection.classList.add("snowy")
+        landscape.innerHTML = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲"
+    }
+}
+
+const updateCityName = () => {
+    const cityInputValue = cityNameInput.value 
+    headerCityName.textContent = cityInputValue
+}
+
+const resetCityName = () => {
+    cityNameInput.value = '';
+    headerCityName.textContent = '';
+}
+
+const skyGardenDisplay = () => {
+    skySection.classList.remove("sunny", "cloudy", "rainy", "snowy");
+    const selectedText = skySelect.value
+
+    if (selectedText === 'sunny') {
+        sky.innerHTML = "☁️ ☁️ ☁️ ☀️ ☁️ ☁️";
+    } else if (selectedText === 'cloudy') {
+        sky.innerHTML = "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️";
+    } else if (selectedText === 'rainy') {
+        sky.innerHTML = "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧";
+    } else if (selectedText == 'snowy') {
+        sky.innerHTML = "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨";
+    }
+}
+const registerEventHandlers = () => {
+    //whatever function is updating UI
+    const increaseButton = document.getElementById('increaseButton');
+    increaseButton.addEventListener("click",temperatureIncrease);
+
+    const decreaseButton = document.getElementById('decreaseButton');
+    decreaseButton.addEventListener("click",temperatureDecrease);
+
+    cityNameInput.addEventListener('input', updateCityName);
+
+    const currentTemperatureButton =  document.getElementById('currentTempButton');
+    currentTemperatureButton.addEventListener("click", getCurrentTemperature)
+
+    skySelect.addEventListener('change', skyGardenDisplay);
+
+    cityNameReset.addEventListener('click', resetCityName);
+
+};
+
+document.addEventListener("DOMContentLoaded", registerEventHandlers);
+
```

Comment:
Nice work ensuring that your document was loaded because adding all your functionality!